docs/user_guide/federated_xgboost/secure_xgboost_user_guide.rst

```diff
@@ -0,0 +1,373 @@
+##########################
+NVFlare XGBoost User Guide
+##########################
+
+Overview
+========
+NVFlare supports federated training with XGBoost. It provides the following advantages over doing the training natively with XGBoost:
+
+- Secure training with Homomorphic Encryption (HE)
+- Lifecycle management of XGBoost processes.
+- Reliable messaging which can overcome network glitches
+- Training over complicated networks with relays.
+
+It supports federated training in the following 4 modes:
+
+1. **horizontal(h)**: Row split without encryption
+2. **vertical(v)**: Column split without encryption
+3. **horizontal_secure(hs)**: Row split with HE (Requires at least 3 clients. With 2 clients, the other client's histogram can be deduced.)
+4. **vertical_secure(vs)**: Column split with HE
+
+When running with NVFlare, all the GRPC connections are local and the messages are forwarded to other clients through CellNet. The local GRPC ports are selected automatically by NVFlare.
```

Comment:
Suggest be more precise:

"... all the GRPC communications in the XGBoost are local"

"... through NVFlare's CellNet communication"